docs/simple-ruby/index.html

```diff
@@ -148,7 +148,7 @@ <h3 id="web-ruby-placement">Web Ruby placement</h3>
 there are also practical limits to how complex the system can be
 in order to be practically implementable.</p>
 
-<p>The following is a proposal for a simple processing system.
+<p>The following is a proposal for a simple processing system. The target audience is implementers.
```

Comment:
s/The target audience is implementers/The target audience is implementers and specification writers/ ?

Reply:
Yes, i was actually going to suggest that. Must have been distracted.

Reply:
Do you want me to make that edit and resubmit, or will you just make after merging?

Reply:
I'd appreciate if you could make the edit.

Reply:
Done.